plugins/inventory/proxmox.py

```diff
@@ -308,12 +308,15 @@ def _get_auth(self):
 
     def _get_json(self, url, ignore_errors=None):
 
-        if not self.use_cache or url not in self._cache.get(self.cache_key, {}):
+        data = []
 
-            if self.cache_key not in self._cache:
-                self._cache[self.cache_key] = {'url': ''}
+        if self.use_cache and not self.update_cache:
+            try: 
```

Comment:
Here's the trailing whitespace that the linter is complaining about:
```suggestion
            try:
```